src/wazuh_modules/vulnerability_scanner/indexer/template/index-template.json

```diff
@@ -16,7 +16,6 @@
         "number_of_shards": "1",
         "query.default_field": [
           "agent.id",
-          "host.os.family",
```

Comment:
Why is this line removed? Is it outside the scope of the issue?

Reply:
It's a leftover from https://github.com/wazuh/wazuh/issues/25485#issue-2498007745